client-tmp/react/src/config.ts

```diff
@@ -0,0 +1,31 @@
+import { RouteLayouts } from './types/RouteLayouts';
+
+export const routeLayouts: RouteLayouts = {
+  '/auth-mock': 'blank',
+  '/auth': 'blank',
+  '/': 'blank',
+};
+
+import type { MainMenu } from './types/menu';
+
+export const AUTH_PATH = 'auth';
+
+export const API_DATA = {
+  URL: 'wss://public-foundation.genesislab.global/gwf/',
```

Comment:
hard-coded just for testing I'm guessing

Reply:
yup, it's not ready to merge / review